react/tic-tac-toe/src/redux/matches/actions.js

```diff
@@ -0,0 +1,20 @@
+import matchesService from '../../services';
+import { GET_MATCHES } from '../../constants';
+
+export const fetchMatches = () => async dispatch => {
+  const matches = await matchesService.getMatches();
```

Comment:
this is fine for now, but normally we dispatch a `GET_MATCHES` action that sets `matchesLoading` to true in the state and some other stuff that we may want to do, and then if `matches.ok` is true we dispatch `GET_MATCHES_SUCCESS`, that sets loading to false and exposes the matches data in the state, and if `matches.ok` is not true we dispatch `GET_MATCHES_FAILURE`, set loading to false, and handle the error in some way. You'll see redux recompose does this automagically.